path.c

```diff
@@ -81,6 +81,7 @@ char **get_search_paths() {
   path[4] = strdup("/usr/share/curseofwar/");
   path[5] = strdup("/usr/share/curseofwar-sdl/");
   path[6] = strdup("/usr/share/curseofwar-common/");
+  path[7] = strdup(INSTALL_DATA);
```

Comment:
Maybe lines 80-83 should be removed (or maybe all the search logic), not sure if they serve a purpose anymore. Unless they are needed for development or something